auth-api/src/main/java/io/stargate/auth/api/impl/Server.java

```diff
@@ -42,12 +44,19 @@
 
 public class Server extends Application<ApplicationConfiguration> {
 
+  private static final String MODULE_NAME = "authapi";
```

Comment:
nice extraction, but maybe the better place for it is an activator class?

Reply:
+1 to move module name constant to the activator.

Reply:
done, resolving.